pnpm-lock.yaml

```diff
@@ -31,9 +33,11 @@ importers:
       '@fluidframework/test-tools': 0.2.3074
       '@microsoft/api-documenter': 7.17.9
       '@microsoft/api-extractor': 7.22.2
+      '@octokit/core': 4.2.0
       c8: 7.7.1
       concurrently: 6.3.0
       copyfiles: 2.4.1
+      danger: 10.9.0_@octokit+core@4.2.0
```

Comment:
Since we can't include a comment about the issue here in this file, at the very least you should note it in the PR description that this is a work around and not a proper fix, and probably include a TODO comment in `build-tools/packages/build-cli/src/commands/run/bundleStats.ts` noting the issue as well.